bleak/backends/client.py

```diff
@@ -46,6 +46,20 @@ def __repr__(self):
 
     # Connectivity methods
 
+    @abc.abstractmethod
+    async def set_disconnect_callback(
```

Comment:
I would like some documentation of an example of a `disconnect_callback` that shows how it is used. Place it here, in the docstring of the interface class.

Reply:
Makes sense. I'll add it tomorrow.